aztk/models/cluster_configuration.py

```diff
@@ -101,7 +101,7 @@ def __validate__(self) -> bool:
             )
 
         if self.custom_scripts:
-            deprecate("Custom scripts are DEPRECATED.", "Use plugins instead. See https://aztk.readthedocs.io/en/v0.7.0/15-plugins.html.")
+            deprecate("0.9.0", "Custom scripts are DEPRECATED.", "Use plugins instead. See https://aztk.readthedocs.io/en/v0.7.0/15-plugins.html.")
```

Comment:
There are several links in the codebase and documentation to v0.7.0 of readthedocs, and also some links to the v0.7.0 tag on GitHub. Should these be updated?

Reply:
yes, we can release a patch update to pitch these issues. Thanks for pointing this out.

Reply:
Maybe it would make sense for them to point to `latest` in the source code, and then to have the release automation fix them to a particular version?